docs/en/manuals/render.md

```diff
@@ -119,6 +119,26 @@ msg.post("@render:", "use_camera_projection")
 ```
 
 
+## Frustum culling
+
+The render API in Defold lets developers perform something called frustum culling. When frustum culling is enabled any graphics that lies outside of a defined bounding box or frustum will be ignored. In a large game world where only a portion is visible at a time, frustum culling can dramatically reduce the amount of data that needs to be sent to the GPU for rendering, thus increasing performance and saving battery (on mobile devices). It is common to use the view and projection of the camera to create the bounding box. The default render script uses the view and projection (from the camera) to calculate a frustum.
+
+Frustum culling is implemented in the engine per component type. Current status (Defold 1.4.7):
+
+| Component   | Supported |
+|-------------|-----------|
+| Sprite      | YES       |
+| Model       | YES       |
+| Mesh        | YES (1)   |
+| Label       | YES       |
+| Particle fx | NO        |
+| Tilemap     | NO        |
+| Spine       | NO        |
```

Comment:
we also have it for spine: https://github.com/defold/extension-spine/pull/106

Reply:
Ah, PR updated!